app/views/posts/_post_with_comments.html.erb

```diff
@@ -2,6 +2,9 @@
   <%= render partial: 'posts/post', locals: { post: post } %>
   
   <div style="border: 1px solid grey; padding:4px;">
+      <% if can? :destroy, @post %>
+      <%= button_to "Delet This Post" ,[@post.author, @post], method: :delete %>
```

Comment:
- [ ] Kindly fix this delete button error on the post index page.

  ![image](https://github.com/bhushan354/Blog-App-Rails/assets/36542986/01218a0f-8739-441d-89c3-5728e6c3cd5a)